File: src/main/java/com/addressbook/dto/OrganizationDto.java
package com.addressbook.dto;

import com.addressbook.model.Organization;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@ToString
@SuppressWarnings("unused")
public class OrganizationDto implements Serializable {

    private String id;
    private String name;
    private String street;
    private String zip;
    private String type;
    private String lastUpdated;

    public OrganizationDto(Organization organization) {
        this.id = organization.getId();
        this.name = organization.getName();
        this.street = organization.getAddr().getStreet();
        this.zip = organization.getAddr().getZip() + "";
        this.type = organization.getType().getEng();
        this.lastUpdated = organization.getLastUpdated().toString();
    }
}

File: addressbook-common/src/main/kotlin/com/addressbook/dto/OrganizationDto.kt
package com.addressbook.dto

import com.addressbook.model.Organization
import java.io.Serializable

class OrganizationDto(id: String?,
                      name: String?,
                      street: String?,
                      zip: String?,
                      type: String?,
                      lastUpdated: String?) : Serializable {

    var id: String? = null
    var name: String? = null
    var street: String? = null
    var zip: String? = null
    var type: String? = null
    var lastUpdated: String? = null

    constructor(organization: Organization?) : this(organization?.id,
            organization?.name,
            organization?.addr?.street,
            organization?.addr?.zip,
            organization?.type?.eng,
            organization?.lastUpdated.toString())

    init {
        this.id = id
        this.name = name
        this.street = street
        this.zip = zip
        this.type = type
        this.lastUpdated = lastUpdated
    }
}
